docs/cli.md

```diff
@@ -12,13 +12,15 @@ NAME:
    config-server-encrypt-value - Encrypt a string using a Spring Cloud Services configuration server
 
 USAGE:
-      cf config-server-encrypt-value CONFIG_SERVER_INSTANCE_NAME VALUE_TO_ENCRYPT
+      cf config-server-encrypt-value CONFIG_SERVER_INSTANCE_NAME [VALUE_TO_ENCRYPT]
 
-Note: if VALUE_TO_ENCRYPT contains characters that are special to the shell, it may be necessary
-to use escape characters or quotes. Refer to the shell's man page for details.
+      NOTE: Either VALUE_TO_ENCRYPT or --file-to-encrypt flag is required, but not both.
 
 ALIAS:
    csev
+
+OPTIONS:
```

Comment:
Has this file been regenerated with the latest help text? It doesn't seem to match what is in `plugin.go`.

Reply:
Done